docs/rancher/07-run-murmuration-services/migrate-es.md

```diff
@@ -44,7 +40,7 @@ kubectl config use-context do-lon1-murmprod
 
 ## Step 2 - Configure Elasticsearch on Source
 
-Execute the configuration script for Elasticsearch and replace `ACCESS_KEY` and `SECRET_KEY` with your own.
+Execute the configuration script for Elasticsearch and replace `ACCESS_KEY` and `SECRET_KEY` with the ones from your bucket.
```

Comment:
I think it's confusing here.

The `ACCESS_KEY` and `SECRET_KEY` should from cloud account credentials.